Software/Python/easygopigo.py

```diff
@@ -0,0 +1,405 @@
+#!/usr/bin/env python
+from __future__ import print_function
+from __future__ import division
+from builtins import input
+
+import sys
+import tty
+import select
+import gopigo
+
+sys.path.insert(0, '/home/pi/Desktop/GoPiGo/Software/Python/line_follower')
```

Comment:
Yes. This can be removed once line_sensor.py is system-wide, and scratch_line.py partially merged into line_sensor.py